packages/@okta/vuepress-site/guides/sign-into-spa/configure-the-sdk/index.md

```diff
@@ -14,9 +14,14 @@ const config = {
   clientId: '{clientId}',
   issuer: 'https://{yourOktaDomain}.com/oauth2/default',
   redirectUri: 'http://localhost:8080/implicit/callback',
+  scope: 'openid profile email',
 };
 ```
 
+> Note: You can also get the issuer config using this link `https://{yourOktaDomain}.com/oauth2/default/.well-known/openid-configuration`
+
+> Note: `openid`, `profile` and `email` are reserved scopes in OpenID Connect to get access to user's data. You can read more about scopes [here](https://developer.okta.com/docs/api/resources/oidc/#scopes)
```

Comment:
This rabbit hole goes deep - was there something specific this link would have helped you with as you went through the tutorial?

Reply:
We didn't have the `scope` config before, which was causing `userName` not being fetched in the UI. So, I added this so that the code works if someone copy pastes from guide.
This note was only for the curious as to why we need the `scope` config. It would only help the user get info as to why we need to pass the `scope` config